### PR TITLE
adding descriptions

### LINKED ISSUE #794: Build project descriptions
First step toward Discover with hashtags is to build out project descriptions. 
- [ ] add description field to settings view
- [ ] display description on project card

@vazquez - let's add annotated comps for just this chunk of work

### PR DESCRIPTION
**Pull request by gvn**
Closes #794

### CODE REVIEW

**Comment by alanmoo** [comment]
Descriptions aren't deployed to the staging API yet, right?

**Comment by xmatthewx** [comment]
Should be there on staging. https://github.com/mozilla/api.webmaker.org/commit/6cba731c9a3807712d14b0dcdff41718975fb74b

**Comment by alanmoo** [comment]
Hmmm...Maybe I did something wrong while testing this, I'll try again.

**Comment by alanmoo** [comment]
Two things I noticed:

When adding/changing a description, going back to the make view does not reflect the change until you manually pull to reload.

When adding a description, there seems to be an issue with saving after the description has been too long. Even after reducing down to a reasonable length, backing out of the view results in the description not saving.

Also take a look at the effect of empty descriptions on the layout of cards, as I mentioned earlier today.

**Comment by gvn** [comment]
I think everything should be working now aside from the whole Make view not updating thing.

File: src/components/card/card.jsx [hunk 1]
```diff
@@ -36,17 +36,23 @@ var Card = React.createClass({
...
 
         <div className="meta">
-          <Link className="avatar" hidden={!this.props.showAuthor} href={"/pages/user-projects/"} url={`/users/${this.props.author.id}/projects`}><img width="47" height="47" src="../../img/avatar-icon.svg" /></Link>
-          <div className="text">
-            <div className="title">{this.props.title}</div>
-            <Link className="author"  hidden={!this.props.showAuthor} href={"/pages/user-projects/"} url={`/users/${this.props.author.id}/projects`}>{this.props.author.username}</Link>
+          <div className="wrapper">
+            <Link className="avatar" hidden={!this.props.showAuthor} href={"/pages/user-projects/"} url={`/users/${this.props.author.id}/projects`}><img width="47" height="47" src="../../img/avatar-icon.svg" /></Link>
+            <div className="text">
+              <div className="title">{this.props.title}</div>
```
Comment (alanmoo) [inline src/components/card/card.jsx:42]: I wonder if we should make the title a link to the project as well...

**Pull request merged**